styles/designs/carnival/parts/_toc-shapes.scss

```diff
@@ -184,3 +184,58 @@
     )),
   )
 ));
+
+//------------------------------------------------------------------------------------------
+// New styles, more consistent with Cosmos
+// (applied only to Organic Chemistry for now)
+
+@include create_shape('carnival:::TocNoUnitShape', (
```

Comment:
I see that there's a `NoUnitToc` above in this file. I'm guessing that, as part of the refactor, you noticed an opportunity to make it better, and that this shape will replace `NoUnitToc` later? 

If that's the case, do you mind adding a comment by `NoUnitToc` to indicate that it's deprecated? Thanks! :)